src/js/components/Graphs/exampleComponent.jsx

```diff
@@ -65,6 +66,7 @@ const exampleGraphDeclaration = {
 
 const ExampleGraph = ({
   pageUrl,
+  tabId,
   // actions,
```

Comment:
Is this needed?

Reply:
this is example component. I would like to show how to use actions for those who never use redux before. Example component will be remove in the future